src/components/NewCommentForm.tsx

```diff
@@ -46,35 +41,34 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
 
   const handleSubmit = async (event: React.FormEvent) => {
```

Comment:
Async is not needed here anymore
```suggestion
  const handleSubmit = (event: React.FormEvent) => {
```